.github/configuration/upgrade-test-configuration.yml

```diff
@@ -30,22 +30,6 @@ upgrade-version: [{
     }
   ]
 },
-{
-  upgrade-path: [
-    {
-      version: 14.3,
-      upgrade-type: null
-    },
-    {
-      version: 14.6,
-      upgrade-type: minor
-    },
-    { 
-      version: target.latest,
-      upgrade-type: major
-    }
-  ]
-},
```

Comment:
Why are we removing this?

Reply:
Just trying to reduce the number of workflows here and since we already have 14.5 -> 14.6 -> latest, I think the above one seems unnecessary.